### PR TITLE
Refactor/server packets

### PR DESCRIPTION
**Pull request by IDragonfire**
Outsource some incoming server packets calls ...
First step to split network layer from ui code ...

Discoveries:
* We have a seperate tournament server
* We have a seperate stats server that use the `handle_*` methods of the client

### CODE REVIEW

**Comment by Sheeo** [comment]
This looks like a step in the right direction, thanks :)

**Comment by IDragonfire** [comment]
If you have suggestion, ideas contact me in slack ...
The next step is to refactore more and more code that we later can change the ui or the server(s) without any problems :D
Next target, Concept for: 
* Client Actions
* Admin Actions
(https://github.com/IDragonfire/lobby/commits/draft/mode_rebased)

File: src/client/server_packets.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from PyQt4 import QtCore
+
+class ServerPackets(QtCore.QObject):
```
Comment (Sheeo) [inline src/client/server_packets.py]: I'd prefer if we gave this class a bit more responsibility than simply being a dictionary with an aliased `.get(x, None)` function. (This is what `translate` essentially does).

I'd suggest calling it CommandDispatcher and to move client.dispatch in here as well.

**Comment by Sheeo** [comment]
Like so?

**Comment by IDragonfire** [comment]
Yes, but we should then name it correctly...  `PrimaryCommandDispatcher ` because we need something similar for the stats and tournaments commands.

**Comment by Sheeo** [comment]
These are going away, so no, we do not :)

**Comment by IDragonfire** [comment]
Then you can merge it from my view

**Comment by Wesmania** [comment]
The messages / signals code has been significantly refactored since this PR, so I'll take the liberty to delete the corresponding PR branch.

**Comment by IDragonfire** [comment]
I stopped python client development, do it.
Some years ago under zep I implemented a friend list and refactored this for the friend list...

**Pull request closed without merging**